CONTRIBUTING.md

```diff
@@ -61,16 +61,14 @@ Pro tip: If any of your breakpoints are in code run by multiple tests, and you r
 
 ## Debug Build Flags
 
-Throughout the codebase, you will find debug flags like `IS_DEBUG_BUILD` guarding various code sections.
+Throughout the codebase, you will find debug flags like `__DEBUG_BUILD__` guarding various code sections.
 These flags serve two purposes:
 
 1. They enable us to remove debug code for our production browser bundles.
 2. Enable users to tree-shake Sentry debug code for their production builds.
 
-These debug flags need to be declared in each package individually and must not be imported across package boundaries, because some build tools have trouble tree-shaking imported guards.
-As a convention, we define debug flags in a `flags.ts` file in the root of a package's `src` folder.
-The `flags.ts` file will contain "magic strings" like `__SENTRY_DEBUG__` that may get replaced with actual values during our, or the user's build process.
-Take care when introducing new flags - they must not throw if they are not replaced.
+Flags will be replaced with booleans for our browser bundles. In cjs and esm builds, they will be replaced by statements that do not throw.
```

Comment:
It won't let me actually make a suggestion here, but if I could, I'd suggest:

```md
Throughout the codebase, you will find the `__DEBUG_BUILD__` flag guarding various code sections. This flag serves two purposes:

1. It enables us to remove debug code from our minified CDN bundles during build, by replacing the flag with `false` before tree-shaking occurs.
2. It enables users to remove Sentry debug code from their production bundles during their own build. When we build our npm packages, we replace the flag with `(typeof __SENTRY_DEBUG__ === 'undefined' || __SENTRY_DEBUG__)`. If the user does nothing, this evaluates to `true` and logging is included. But if the user runs their own replacement during build (again replacing the flag with `false`), the build will tree-shake the logging away, just as our bundle builds do.

Note that the replacement flag, `__SENTRY_DEBUG__`, is different from the original flag . This is necessary because the replacement plugin runs twice, at two different stages of the build, and we don't want to run a replacement on our replacement (as would happen if we reused `__DEBUG_BUILD__`).
```

Reply:
Great change. Thank you! a4b07d329c142b5462c276de7f4da2a9449e961a